proto/system_config.proto

```diff
@@ -126,6 +126,9 @@ message DaqConfig {
 
   // Drop into console mode after test
   bool use_console = 52;
+
+  // Port of the server to receive device result
+  bool device_result_port= 53;
```

Comment:
- bool?  Shouldn't this be an int?
- Space before =
- Encapsulate in submessage (e.g. something new similar to RunTrigger). I.e., what if we (in the future) also wanted to specify an ip address?

Reply:
Encapulsated in DeviceReporting.